### ISSUE TITLE
Train new parser model for spanish

### ISSUE THREAD

**Issue by sersoage** [explosion/spaCy#2018]
Hi I am working to improve the spanish model (i could provide the model after training if you want). I checked the training examples and saw that in that case a blank model is created. Is this recommended? Could the production model be used as pre training? if this is the case, how does it handle new words? If you can point me to the right direction I would appreciate.
Thanks

**Comment by honnibal** [comment]
@sersoage You could try starting from the pre-trained model, although I haven't really done that much.

If I have enough examples to train from scratch, I would usually prefer that. Starting from a pre-trained model makes your experiments more difficult to reason about. Hal Daume has a nice post on this: https://nlpers.blogspot.de/2016/05/a-bad-optimizer-is-not-good-thing.html

An alternative to starting from the pre-trained model is using it to label data, and then using those as part of your training set.
